source/_posts/2024-02-21-voice-chapter-6.markdown

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: "Voice - Chapter 6"
+description: "Improved errors and ESPHome wake word"
+date: 2024-02-21 00:00:00
+date_formatted: "February 21, 2024"
+author: Michael Hansen
+comments: true
+categories: Assist
+og_image: /images/blog/2024-02-21-voice-chapter-6/social.jpg
```

Comment:
The are of the midjourney generated pictures for Chapters is over?

Reply:
![image](https://github.com/home-assistant/home-assistant.io/assets/5878296/d80ade43-9605-44e1-9ac3-6190a5fa3a98)

(In case it is not ;) )